utils/node_local_test.py

```diff
@@ -671,7 +671,8 @@ def start(self):
         cmd = ['storage', 'format', '--json']
         while True:
             try:
-                self._sp.wait(timeout=0.5)
+                rc = self._sp.wait(timeout=0.5)
+                print(rc)
```

Comment:
Maybe a more explicit message could be helpful for the end user.
I am also supposing that this is some old debugging traces which have not yet been removed?

Reply:
@ashleypittman , please could you tell me (just for my understanding), if this print is used for debugging purpose or if I miss something.
Thanks in advance.

Reply:
It's debugging.  What it prints is the exit code of daos_server if it's exited, if this is -4 then we can tell that the exit was because of SIGILL which is the spdk build issue.